### PR TITLE
Add healthcheck and test files

### PR DESCRIPTION
**Pull request by WaylonOKC**

### CODE REVIEW

File: test/setup.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+// do not load env config in "CI" mode, i.e., when Heroku is running tests
+if (!process.env.CI) {
+  require("dotenv").config({ path: ".env.test" });
+}
+
+// use the HerokuRedis AddOn `REDIS_URL` var
+// as the value for the unique `ENGINE_REDIS_URL` var
+if (!process.env.ENGINE_REDIS_URL && process.env.REDIS_URL) {
+  process.env.ENGINE_REDIS_URL = process.env.REDIS_URL;
+}
```
Comment (markhealey) [inline test/setup.js:11]: Pretty sure you can remove this code (lines 7-11)

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 
 node_modules/
 .env
-public/dist/
\ No newline at end of file
+public/dist/
+.env.test
```
Comment (markhealey) [inline .gitignore:5]: Just change `.env` to `.env*`

File: .mocharc.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	asyncOnly: true,
+	recursive: true,
+	reporter: "spec",// dot, spec, nyan
+	timeout: 5000
+}
```
Comment (markhealey) [inline .mocharc.js:6]: It's obvious in this file but present in the others you added as well — you need to press `CMD+SHIFT+P` then type `Convert indentation to spaces` to ditch the tab spacing.

File: app.js [hunk 4]
```diff
@@ -32,6 +33,7 @@ if (isProduction) {
 // view engine setup
 app.set("views", path.join(__dirname, "views"));
 app.set("view engine", "ejs");
+app.set("pkg", pkg);
```
Comment (markhealey) [inline app.js:36]: Don't need this if you're using `const pkg = require("../package.json");` in the `healthcheck` file

**Comment by WaylonOKC** [comment]
[Setup Healthcheck for Showcase](https://trello.com/c/G0LwTwy2/2727-setup-healthcheck-for-showcase)

**Pull request merged**